src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/AvoidStaticImportCheckTest.java

```diff
@@ -50,17 +50,17 @@ public void testDefaultOperation()
         final DefaultConfiguration checkConfig =
             createModuleConfig(AvoidStaticImportCheck.class);
         final String[] expected = {
-            "23: " + getCheckMessage(MSG_KEY, "java.io.File.listRoots"),
-            "25: " + getCheckMessage(MSG_KEY, "javax.swing.WindowConstants.*"),
-            "26: " + getCheckMessage(MSG_KEY, "javax.swing.WindowConstants.*"),
-            "27: " + getCheckMessage(MSG_KEY, "java.io.File.createTempFile"),
-            "28: " + getCheckMessage(MSG_KEY, "java.io.File.pathSeparator"),
-            "29: " + getCheckMessage(MSG_KEY, "java.lang.Math.E"),
-            "30: " + getCheckMessage(MSG_KEY, "java.lang.Math.sqrt"),
-            "31: " + getCheckMessage(MSG_KEY,
+            "23:27: " + getCheckMessage(MSG_KEY, "java.io.File.listRoots"),
+            "25:42: " + getCheckMessage(MSG_KEY, "javax.swing.WindowConstants.*"),
+            "26:42: " + getCheckMessage(MSG_KEY, "javax.swing.WindowConstants.*"),
```

Comment:
This is outside the scope of the issue, but please - this import in file and duplicated violation, please remove them.

Reply:
Done